blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,87 @@
+
+#### Broader region availability
+Core Kubernetes extensions are available in all regions across all clouds (Public cloud, Fairfax, Mooncake, and Air Gapped cloud). See the full list at [Azure Arc enabled Kubernetes region support](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=azure-arc&regions=all).
+
+#### Native AKS experience
+* Transitioning Add-ons to Core Extensions: Some AKS add-ons, such as Container Insights, Azure Managed Prometheus, and App Monitoring, are transitioning to core extensions. The user experience remains unchanged, but the underlying technology is shifting to extensions.
+
+    CLI command to enable a core Extension that transitioned from an add-on:
+    ```bash
+    az aks create/update --resource-group <group> --name <cluster> --enable-<core-extension>
+    ```
+* Shifting Regular Extensions to Core Extensions: Key extensions like Azure Backup are also being shifted to core extensions for better integration with AKS.
+
+    CLI commands:
+    * Create a regular extension:
+    ```bash
+    az k8s-extension create --extension-type <type> --resource-group <group> --cluster-name <name> --cluster-type <clusterType> --name <extension name>
+    ```
+    * Create a core extension:
+    ```bash
+    az aks extension create --extension-type <type> --resource-group <group> --cluster-name <name> --name <core extension name>
+    ```
+
+#### Safer version management rules
+Core extensions follow the same update rules as add-ons: patch versions can be upgraded within a Kubernetes minor version, while major/minor upgrades occur only with Kubernetes minor version updates to avoid introducing breaking changes in a Kubernetes version.
+
+#### Improved security and efficiency
+
+ExtensionManager is moved from user node pool to AKS Control Plane, which improved extension security and efficiency.
+
+What Is ExtensionManager?
+
+ExtensionManager is a key component that handles lifecycle operations (create, upgrade, delete, reconcile) for Kubernetes extensions. It periodically polls for version updates on extension instances and reconciles extension configuration settings, ensuring that extensions are managed effectively. Previously running on customer worker nodes, it has now been moved to the AKS control plane.
+
+What Has Changed?
+
+Previously, extension-operator and extension-agent were deployed in the kube-system namespace by ExtensionManager on worker nodes. These components are now part of the AKS control plane, so they are no longer visible in the kube-system namespace.
```

Comment:
I'd stick to either calling the pods out directly (`extension-operator` and `extension-agent`) or calling it ExtensionManager. Don't switch between the two, because it's sorta confusing.

Reply:
i.e. here you could say "previously, ExtensionManager pods were deployed in the kube-system namespace.